apps/web/src/server/applications/applications.types.d.ts

```diff
@@ -89,3 +89,16 @@ export interface CaseCreateProps<BodyValues> {
 	values: BodyValues;
 	layout?: FormCaseLayout;
 }
+
+export interface DocumentationFile {
+	fileName: string;
+	url: string;
+}
+
+export interface DocumentationPageProps {
```

Comment:
better move this DocumentationPageProps in a documentation/documention.types.d.ts as it's specific of a very small portion of the code

Reply:
yes I wondered about this myself, good idea!